src/wagtailtrans/models.py

```diff
@@ -139,39 +139,116 @@ def _language_default():
         return default_language.pk
 
 
-class TranslatablePage(Page):
-
-    #: Defined with a unique name, to prevent field clashes..
-    translatable_page_ptr = models.OneToOneField(Page, parent_link=True, related_name='+', on_delete=models.CASCADE)
+class TranslatablePageItem(models.Model):
+    page = models.ForeignKey(
+        "wagtailcore.Page",
+        related_name="translatable_page_item",
```

Comment:
@nnist ForeignKey reverse set could in theory have more, so translatable_page_items (plural) makes sense here. Or see if we can use this field as a OneToOneField.